### PR TITLE
remove inkeeper

### PR DESCRIPTION
**Pull request by szuecs**
remove inkeeper https://github.com/zalando/innkeeper that was never really in production and abandoned 5y ago

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -774,20 +774,16 @@ func (c *Config) ToOptions() skipper.Options {
...
-		EtcdPrefix:                c.EtcdPrefix,
-		EtcdWaitTimeout:           c.EtcdTimeout,
-		EtcdInsecure:              c.EtcdInsecure,
-		EtcdOAuthToken:            c.EtcdOAuthToken,
-		EtcdUsername:              c.EtcdUsername,
-		EtcdPassword:              c.EtcdPassword,
-		InnkeeperUrl:              c.InnkeeperURL,
-		InnkeeperAuthToken:        c.InnkeeperAuthToken,
-		InnkeeperPreRouteFilters:  c.InnkeeperPreRouteFilters,
-		InnkeeperPostRouteFilters: c.InnkeeperPostRouteFilters,
```
Comment (AlexanderYastrebov) [inline config/config.go:787]: I thin there are flags for these that still need to be removed

Comment (szuecs) [reply config/config.go:787]: Yes, thanks some todo

Comment (szuecs) [reply config/config.go:787]: done

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**